### DISCUSSION TITLE
Make previously submitted data available to fields in multi step form.

### DISCUSSION THREAD

**Discussion by jaandrews** [umbraco/Umbraco.Forms.Issues#1505] [Ideas]
Basically, it would be handy to have a built in way to access data that has been previously submitted in a multi step before the user submits. There are a couple potential use cases for such a feature.

1. A review field that displays a summary of what the user has submitted.
2. Modify options based on a previous entry. For example if a user was prompted to select a country in the first step and a state in the second, the states could be filtered to only include states from that country.

**Comment by AndyButland** [comment]
Thanks for the suggestion - and yes, there are a few things around multi-step forms we'd like to improve, and will hopefully get a chance to in the upcoming months.

See also #281

**Comment by AndyButland** [comment]
We'll be adding a review page in the next minor release of Forms.  It doesn't fully address what this issue was raised for but will provide a means of providing this summary of what the user has completed, before they submit their form.

**Comment by TvGessel** [comment]
We've reviewed this, but as we don't see a broad, generic use case for this feature at the moment so it isn't something we plan to pick up.
